app/views/order_details/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<div>
```

Comment:
order detail thì nên hiển thị theo từng order thì hợp lý hơn chứ em nhỉ?
em hiển thị list order detail mục đích làm gì?

Reply:
Trang này e dùng để hiện thị sản phẩm mà ng dùng đã thêm vào giỏ hàng mà chưa thanh toán ạ chắc e nhầm nền chắc e fix lại sau ạ